### DISCUSSION TITLE
Installation Problem

### DISCUSSION THREAD

**Discussion by giosax** [retorquere/zotero-better-bibtex#1851] [Q&A] [answered]
Hi, I'm Giovanni from Italy.
I would like to ask the experts for help because I cannot install Better BibTex.
After downloading the latest version, I select Tools / Add On, but I cannot find the xpi file in the downloaded folder.

Alternatively, I tried to run the "install.rdf" file from the finder (I'm a mac user) but zotero does not recognize the file saying that it is not compatible.

Where am I wrong?

Thanks for the help you can give me.
Have a nice day!
Giovanni

**Comment by retorquere** [comment, marked as answer]
You don't need the install.rdf (where was it offered for download?), the xpi is the right file, but I can't know where you downloaded it to.

**Reply by giosax** [reply]
Thanks for your reply.
Now I understand where I was wrong: Safari, my browser, "reads" the xpi file as a zip file, then automatically decompresses it.
Basically: I was trying to install the unzipped folder, it was my mistake.
Now that I understood the mistake, I was able to do everything.
Thanks once again!

**Comment by fabiez** [comment]
Hi, I installed BBT v6.7.211 on Zotero 6.0.37 on Mac. Unfortunately the add-on fails in connecting to server. 
Zotero suggests this page info: https://www.zotero.org/support/kb/connection_error However, I don't have much knowledge to progress further and fix the issue: any suggestions?
Many thanks!
F

**Reply by fabiez** [reply]
I actually installed the previous version zotero-better-bibtex-6.7.210.xpi and it works!

**Reply by retorquere** [reply]
Please open an issue. Bug diagnosis happens in issues, not discussions.

**Reply by retorquere** [reply]
Also your issue has nothing to do with the current discussion, which concerned not being able to install BBT at all.

**Reply by retorquere** [reply]
Posting that 210 works for you but 211 doesn't isn't really helpful without followup. There's nothing I can do if I don't at least get a log. Please open an issue.
